scripts/Update-QDKVersion.ps1

```diff
@@ -18,7 +18,8 @@ param(
         
     .EXAMPLE
     
-        PS> ./Update-QDKVersion.ps1 -Version 0.11.2003.3107
+        PS> ./Update-QDKVersion.ps1 -Version 
```

Comment:
Please the version number in the example, without the version this command doesn't work (at least on my machine). You could use the latest version here instead of the old one, `0.12.20072031`

Reply:
Please also update the script name in the update instructions at https://github.com/microsoft/QuantumKatas/blob/master/.github/CONTRIBUTING.md to match the new name